docs/pages/application-access/enroll-kubernetes-applications/get-started.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Get Started with Kubernetes Application Discovery
+description: Detailed guide for configuring Kubernetes Application Discovery.
+---
+
+Teleport can automatically detect applications running in your Kubernetes
+clusters and register them with your Teleport cluster. In this setup, users with
+Kubernetes-hosted infrastructure can configure secure access to any new
+applications they deploy with no need for manual intervention beyond the initial
+setup step.
+
+In this guide, we show you how to enable Kubernetes application auto-discovery.
+
+## Prerequisites
```

Comment:
@AntonAM @ptgott 
Can we have a migration section from an installation without the app and discovery services to this new thing?